### ISSUE TITLE
[Bug] Serverlist not refreshing from Node System after initial refresh

### ISSUE THREAD

**Issue by Dss0** [iw4x/iw4x-client#6]
**What are you trying to do?**
Refresh the Serverlist again after it initially fell back to the node system and loaded servers.

**What problem are you having?**
Serverlist does not populate again nor does the "falling back to node system" message show.

**What version of IW4x are you using?**
latest

**Comment by Dss0** [comment]
It actually does refresh when you hit refresh after the initial population but it takes 2 minutes before it falls back to the node system.

**Comment by florisdipt** [comment]
If I am correct this needs updating.

https://github.com/iw4x/iw4x-client/blob/4b1aade8994dcca25bdee54015046d82dcf5134f/src/Components/Modules/ServerList.cpp#L896

There is already some others that have created a server browser.
http://api.raidmax.org:5000/servers#IW4

https://minopia.de/iw4/servers/?games=iw4

But I have not been able to use them yet even by setting the "masterServerName" via console.

Is the source code of the master server available anywhere but I wasn't able to find it anywhere

**Comment by Dss0** [comment]
There is no master server, iw4x uses a decentralized network to share a list of servers between clients/servers. You can't simply set the master server dvar to some random website that displays iw4 servers. RaidMax' site populates that list using iw4madmin (a iw4x server administration software), so only servers running iw4madmin will be displayed there. The minopia list seems to be harcoded as many of those servers are on ancient versions of iw4x and/or don't even exist anymore.
That being said, you don't need to do that either. The serverlist works just fine (with the restriction this issue is about) and displays all the servers that can be seen on those websites as well.

**Comment by diamante0018** [comment]
@Dss0 node system was again put as a secondary business so I don't know about this issue. Feel free to decide whenever you want to continue to pursue this or not

**Comment by Dss0** [comment]
well it's low priority now but it should still be fixed at some point imo.

**Comment by geckotdf** [comment]
In the Client it's possible to disable the Node System searching?

**Comment by diamante0018** [comment]
Nope. It would be an horrible idea. If the master server went down the fall back to node must trigger automatically and immediately for ease of use

**Comment by geckotdf** [comment]
Perfect, What ports need to be open for the Node system to work?

**Comment by diamante0018** [comment]
As a client you need an open internet connection. As a server the net_port needs to be open

**Comment by diamante0018** [comment]
The issue seems to be fixed on the latest IW4x @Dss0 

Almost wanna bet the menu refactor helped

https://youtu.be/n5UUJ8C_rO8

**Comment by mxve** [comment]
I had a similar experience ![image](https://github.com/user-attachments/assets/e6cbc97e-7227-4032-a86d-8040edd143c2)
